### ISSUE TITLE
Does learning rate apply to linear booster?

### ISSUE THREAD

**Issue by david-cortes** [dmlc/xgboost#11106]
There are some hints throughout the R examples for linear boosters, such as in the docs for `xgb.cb.gblinear.history`, which mention things like:
> With the deterministic coordinate descent updater, it is safer to use higher learning rates.

https://github.com/dmlc/xgboost/blob/543b57f33243219eaa428caa1693436c24e7d914/R-package/R/callbacks.R#L984

The examples then pass `learning_rate` (previously `eta`) in the parameters:
https://github.com/dmlc/xgboost/blob/543b57f33243219eaa428caa1693436c24e7d914/R-package/R/callbacks.R#L990

.. but according to the docs about parameters, `eta` and `learning_rate` are for **tree-based** boosters, or at least that's how they are documented, since they aren't listed as parameters for the linear booster:
https://github.com/dmlc/xgboost/blob/543b57f33243219eaa428caa1693436c24e7d914/doc/parameter.rst?plain=1#L78

Does the linear booster use `learning_rate` / `eta` if passed under `params`?

**Comment by trivialfis** [comment]
Yes, it's used.

**Comment by trivialfis** [comment]
https://github.com/dmlc/xgboost/blob/543b57f33243219eaa428caa1693436c24e7d914/src/linear/updater_coordinate.cc#L57

**Comment by david-cortes** [comment]
Thanks. Sounds like the docs could be updated then, so as to list `learning_rate` under parameters for linear booster too. Other parameters like `lambda` are listed under both "tree booter" and "linear booster".

**Comment by trivialfis** [comment]
Thank you for updating the doc!

**Comment by david-cortes** [comment]
@trivialfis But I only updated the R docs. The online docs are still not showing the parameter under "linear booster". It's also not clear to me whether the default value is the same for both booster types either.
